loopd/swapclient_server.go

```diff
@@ -580,14 +585,40 @@ func (s *swapClientServer) ListSwaps(ctx context.Context,
 			continue
 		}
 
-		rpcSwap, err := s.marshallSwap(ctx, &swp)
+		rpcSwapInfo = append(rpcSwapInfo, &swp)
+	}
+
+	// Sort the swaps by initiation time in ascending order (oldest first).
+	slices.SortFunc(rpcSwapInfo, func(a, b *loop.SwapInfo) int {
+		return cmp.Compare(a.InitiationTime.UnixNano(), b.InitiationTime.UnixNano())
```

Comment:
This line is too long. Please format it like this:
```go
return cmp.Compare(
  a.InitiationTime.UnixNano(),
  b.InitiationTime.UnixNano(),
)
```

Reply:
sorry, thanks for continuing to point these out.